Application/mobile/app_code/lib/utils/time.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:intl/intl.dart';
+
+String formatDateTime(String dateTimeString) {
+  DateTime dateTime = DateTime.parse(dateTimeString);
+  return DateFormat('dd/MM/yyyy HH:mm').format(dateTime);
```

Comment:
I think you should use the Format given by the Locale. Since for example, in the US, the format is MM/dd/yyyy)